src/web/mjs/connectors/JpMangas.mjs

```diff
@@ -0,0 +1,14 @@
+import MangaReaderCMS from './templates/MangaReaderCMS.mjs';
+
+export default class JpMangas extends MangaReaderCMS {
+
+    constructor() {
+        super();
+        super.id = 'jpmangas';
+        super.label = 'JpMangas';
```

Comment:
May use same spelling as on website

![image](https://user-images.githubusercontent.com/27136773/96598171-e34e2d80-12ee-11eb-8e52-df8113c3d6e1.png)

Reply:
im either blind or that is the same spelling

Reply:
figured it out